deployments/kubernetes/chart/reloader/templates/_helpers.tpl

```diff
@@ -28,6 +28,21 @@ heritage: {{ .Release.Service | quote }}
 app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
 {{- end -}}
 
+{{/*
+Create pod anti affinity labels
+*/}}
+{{- define "reloader-podAntiAffinity" -}}
+podAntiAffinity:
+  preferredDuringSchedulingIgnoredDuringExecution:
+  - labelSelector:
```

Comment:
I think there are issues here. When I try installing the chart with helm, it gives an error.
```bash
Error: INSTALLATION FAILED: unable to build kubernetes objects from release manifest: error validating "": error validating data: [ValidationError(Deployment.spec.template.spec.affinity.podAntiAffinity.preferredDuringSchedulingIgnoredDuringExecution[0]): unknown field "labelSelector" in io.k8s.api.core.v1.WeightedPodAffinityTerm, ValidationError(Deployment.spec.template.spec.affinity.podAntiAffinity.preferredDuringSchedulingIgnoredDuringExecution[0]): unknown field "topologyKey" in io.k8s.api.core.v1.WeightedPodAffinityTerm, ValidationError(Deployment.spec.template.spec.affinity.podAntiAffinity.preferredDuringSchedulingIgnoredDuringExecution[0]): missing required field "weight" in io.k8s.api.core.v1.WeightedPodAffinityTerm, ValidationError(Deployment.spec.template.spec.affinity.podAntiAffinity.preferredDuringSchedulingIgnoredDuringExecution[0]): missing required field "podAffinityTerm" in io.k8s.api.core.v1.WeightedPodAffinityTerm]
```
As per the k8s documentation [here](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#an-example-of-a-pod-that-uses-pod-affinity)
It should be like this

```yaml
podAntiAffinity:
  preferredDuringSchedulingIgnoredDuringExecution:
  - weight: 100
    podAffinityTerm:
      labelSelector:
        matchExpressions:
        - key: app
          operator: In
          values:
          - {{ template "reloader-fullname" . }}
      topologyKey: "kubernetes.io/hostname"
```

Reply:
This is fixed with my latest commits